db/forward_iterator.h

```diff
@@ -86,6 +86,10 @@ class ForwardIterator : public InternalIterator {
  private:
   void Cleanup(bool release_sv);
   void SVCleanup();
+  static void SVCleanup(
```

Comment:
Why not make this a static function inside forward_iterator.cc instead of member function? Because ForwardIterator is a friend of DBImpl, and this function accesses some private method of DBImpl.